src/backers.ts

```diff
@@ -14,17 +14,23 @@ export function getCollectiveBackerTiers(
   allBackers: Member[],
   backerName: string,
   backerOrganisations: string[],
-): Tier[] {
-  const tiers = allBackers.reduce<Tier[]>((acc, member) => {
-    if (!member.github) return acc
+): Tier[] | null {
+  const tiers = allBackers.reduce<Tier[] | null>((acc, member) => {
+    if (!member.github || acc === null) return acc
 
     const githubName = stripGithubName(member.github)
     const isMember = [...backerOrganisations, backerName].some(is(githubName))
 
+    /* Tiers */
```

Comment:
In the case of @Betree, he was an 'ADMIN' and a 'BACKER' at the same time. For that reason I would check ADMIN before checking tiers.

Reply:
It doesn't matter because once it hits `null` it stays `null`; therefore, there's no difference in the order. Good catch, by the way, I had to double check if I had right thinking process.

Reply:
It is this way because of the early exit right? `if (!member.github || acc === null) return acc`

Reply:
yup, exactly 🙂